src/isomorphic/classic/element/ReactCurrentOwner.js

```diff
@@ -11,20 +11,6 @@
 
 'use strict';
 
-/**
- * Keeps track of the current owner.
- *
- * The current owner is the component who should own any components that are
- * currently being constructed.
- */
-var ReactCurrentOwner = {
-
-  /**
-   * @internal
-   * @type {ReactComponent}
-   */
-  current: null,
-
-};
+var ReactCurrentOwner = require('react-current-owner');
```

Comment:
This won't work out of the box in our primary `@providesModule` world. We could make it but it's trickier and more prone to breakage.